04 - Guides, Workflows, & Courses/Guides/Best Practices and Tips for Theme Development

```diff
@@ -0,0 +1,14 @@
+---
+aliases: 
+- 
+tags:
+- seedling
+publish: true
+---
+
+# Best Practices and Tips for Theme Development
+
+This note serves as a collection of tips and best practices for theme development which are too small to be their own guide.
+
+- By naming your base64-encoded font-faces (`@font-face`) the same as the font you installed on your system, you will never actually see how the fonts are rendered for other users, since Obsidian will prefer the system font when there are multiple fonts with the same name available (one via base64-encoded CSS, one on the system). To prevent potential issues with fonts from going unnoticed, you should name the font-faces in your theme's CSS differently than the one you installed on your system.
```

Comment:
With a suggestion as verbose as this, I recommend the tip be at the top of the paragraph, not the bottom. Then the rest can be an explanation as to why you should follow it:

```suggestion
- Name the font-faces in your theme's CSS different from the one you installed on your system to prevent potential issues with fonts from going unnoticed. 
    By naming your base64-encoded font-faces (`@font-face`) the same as the font you installed on your system, you will never actually see how the fonts are rendered for other users, since Obsidian will prefer the system font when there are multiple fonts with the same name available (one via base64-encoded CSS, one on the system).
```

Reply:
Could there be a concrete example of this renaming? And what effect will it have on users of your theme?

Reply:
@claremacrae None on the users really, it's mostly a debugging issue.

I think @SlRvb's rephrasing if fine